styles.css

```diff
@@ -49,155 +31,376 @@ body {
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
   color: #344563;
-  flex: none;
-  order: 1;
-  flex-grow: 0;
+  margin: 12px 24px 0 24px;
 }
 
 .follow {
-  margin: 20px 0 0 0;
-  width: 117px;
-  height: 24px;
   font-family: "Poppins", sans-serif !important;
   font-style: normal;
   font-weight: 500;
   font-size: 16px;
   line-height: 24px;
   color: #7f8cff;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
+  margin: 10px 24px 0 24px;
 }
 
-.list {
-  height: 30px;
-  width: 327px;
-  align-items: flex-start;
+.social-media-icon-container {
+  display: flex;
 }
 
-.list-item {
-  flex: 1 10fr;
-  width: 20px;
+.social-media-icon {
+  color: #505f79;
+  height: 20px;
+  margin: 10px 15px 136px 24px;
 }
 
-.fa {
+.social-media-icon-color {
   color: #505f79;
 }
 
-.fa:hover {
-  color: #505f795c;
+.grid-item:nth-child(2),
+.grid-item:nth-child(3),
+.grid-item:nth-child(4),
+.grid-item:nth-child(5) {
+  height: 552px;
+  margin: 0 24px 0 24px;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: flex;
+}
+
+.grid-item:nth-child(3),
+.grid-item:nth-child(5) {
+  height: 596px;
+}
+
+.grid-item-two-header,
+.grid-item-three-header {
+  margin: 16px 16px 0 16px;
+}
+
+.grid-item-two-body-title {
+  color: #172b4d;
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 16px 0 0 16px;
+  width: 220px;
+}
+
+.grid-item-two-body-subhead {
+  color: #344563;
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  margin: 20px 0 0 16px;
+}
+
+.subhead-title-year {
+  display: flex;
+}
+
+.counter-image {
+  margin: 20px 12px 0 12px;
+}
+
+.title-year {
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  margin: 20px 0 0 0;
+}
+
+.grid-item-two-body-text {
+  color: #344563;
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  margin: 20px 16px 0 16px;
+}
+
+.js-css-html-button-ul {
+  display: flex;
+  margin: 15px 0 0 0;
+}
+
+.js-css-html-button-li {
+  margin: 0 0 0 16px;
 }
 
-.ellipse {
-  position: absolute;
-  width: 105px;
-  height: 88px;
-  left: 275px;
-  top: 152.24px;
-  transform: rotate(-10deg);
+.html-css-js-button {
+  background-color: #ebebff;
+  border-radius: 8px;
+  color: #6070ff;
+  border: 0;
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
 }
 
-.pie-shape {
-  position: absolute;
-  width: 38.5px;
-  height: 19.25px;
-  left: 16.04px;
-  top: 161.47px;
+.see-project-text {
+  border: 1px solid #396df2;
+  border-radius: 10px;
+  letter-spacing: 0.03em;
+  height: 50px;
+  font-family: "Poppins", sans-serif !important;
```

Comment:
- [Optional] I see that you declare the `font-family`property with the same value all over your style sheet. You should consider creating a class for it rather than repeating yourself. For example, you can create a class `.font { font-family: "Poppins", sans-serif }` and give all the HTML elements that need to have this property the class of `font`. You can read more about DRY code [here](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

Reply:
@Kamba56 Hi I have noted the changes. I will implement them before merging. Thank you.